src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2.java

```diff
@@ -20,78 +20,155 @@
+ *     (iii) Mutect2 also differs from the HaplotypeCaller in that it can apply various prefilters to sites and variants depending on the use of
+ *     a matched normal (--normalSampleName), a panel of normals (PoN; --normal_panel) and/or a common population variant resource containing allele-specific frequencies (--germline_resource).
+ *     If provided, Mutect2 uses the PoN to filter sites and the germline resource and matched normal to filter alleles.
+ *     (iv) Mutect2 variant site annotations differ from that of HaplotypeCaller. See the --annotation parameter description for a list.
+ *     (v) Finally, Mutect2 has additional parameters not available to HaplotypeCaller that factor in the decision to reassemble a genomic region,
+ *     factor in likelihood calculations that then determine whether to emit a variant, or factor towards filtering.
+ *     These parameters include the following and are each described further in the arguments section.
+ * </p>
+ *
+ * <dl>
+ *     <dd>--min_variants_in_pileup ==> active region determination</dd>
+ *     <dd>--minNormalVariantFraction ==> active region determination</dd>
+ *     <dd>--tumorStandardDeviationsThreshold ==> active region determination</dd>
+ *     <dd>--af_of_alleles_not_in_resource ==> germline variant prior</dd>
+ *     <dd>--log_somatic_prior ==> somatic variant prior</dd>
+ *     <dd>--normal_lod ==> filter threshold for variants in tumor not being in the normal, i.e. germline-risk filter</dd>
+ *     <dd>--tumor_lod_to_emit ==> cutoff for tumor variants to appear in callset</dd>
+ * </dl>
+ *
+ * <p>
+ *     Additional parameters that factor towards filtering, including normal_artifact_lod (default threshold 0.0) and
+ *     tumor_lod (default threshold 5.3), are available in {@link FilterMutectCalls}. While the tool calculates
+ *     normal_lod with a fixed ploidy assumption given by the --sample_ploidy option (default is 2), it calculates
+ *     normal_artifact_lod with the same approach it uses for tumor_lod, i.e. with a variable ploidy assumption.
+ * </p>
+ *
+ * <p>
+ *     If the normal artifact log odds becomes large, then FilterMutectCalls applies the artifact-in-normal filter.
+ *     For matched normal samples with tumor contamination, consider increasing the normal_artifact_lod threshold.
+ * </p>
+ *
+ * <p>
+ *     The tumor log odds, which is calculated independently of any matched normal, determines whether to filter a tumor
+ *     variant. Variants with tumor LODs exceeding the threshold pass filtering.
+ * </p>
+ *
+ * <h3>Examples</h3>
+ *
+ * <p>Example commands show how to run Mutect2 for typical scenerios.</p>
+ *
+ * <h4>Tumor with matched normal</h4>
+ * <p>
+ *     Given a matched normal, Mutect2 is designed to call somatic variants only. The tool includes logic to skip
+ *     emitting variants that are clearly germline based on the evidence present in the matched normal. This is done at
```

Comment:
We on the GATK team are insiders who use "germline" as an adjective, but for the standard user I might use more explicit language like "present in the germline".